frontend/src/components/pages/CampOverview/CampersTable/WaitlistedCampersTable.tsx

```diff
@@ -83,10 +90,71 @@ const WaitlistedCampersTable = ({
     }
   };
 
+  const deleteWaitlistedCameper = async (
+    waitlistedCamper: WaitlistedCamper,
+  ) => {
+    setCamperToDelete(waitlistedCamper);
+    onOpen();
+  };
+
+  const confirmDeleteWaitlistedCamper = async (
+    waitlistedCamper: WaitlistedCamper | null,
+  ) => {
+    if (waitlistedCamper) {
```

Comment:
maybe in very rare edge cases - the null type was more to accommodate for the prop being passed in since it's initialized as null in the useState but will always be set to a waitlisted camper if the user opens the modal using the context menu